server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSLexer.g4

```diff
@@ -660,6 +661,7 @@ PURGEABILITY : P U R G E A B I L I T Y;
 PURGEABLE : P U R G E A B L E;
 PUSH : P U S H;
 PUT : P U T;
+PUT64: P U T '6' '4';
```

Comment:
Should these new tokens be added to the `cicsLexerDefinedVariableUsageTokens` rule?

Reply:
Done.